README.md

```diff
@@ -1 +1,283 @@
+
+```shell
+$ docker compose up
+```
+
+## Mandatory Part
+
+Your task is to rewrite the existing endpoints in the **hot-coffee** project to work with a PostgreSQL database. To do this, you may use a third-party PostgreSQL driver as part of your solution.
+
+You need to create the following tables: `user`, `order`, `menu`, `menu_item`, `inventory`, and `ingredient`, and define the relationships between them. You are not limited to only these tables; you may create additional ones if necessary for implementing the required functionality. The design and relationships of the tables are at your discretion.
+
+Pay special attention to designing the relationships between tables (e.g., one-to-many, one-to-one). For instance, the `menu` and `menu_item` tables will have a "one-to-many" relationship, meaning the `menu_item` table should include a column with the following constraint:
+```sql
+menu_id INT NOT NULL CONSTRAINT fk_menu FOREIGN KEY(menu_id) REFERENCES menu(id)
+```
+Plan your data structure carefully to ensure the project's scalability and maintainability.
+
+Since adding a database dependency to your project makes it more challenging for auditor to run and test it, you need to containerize your service and database into separate containers and write a `docker-compose.yml` file. This will allow the project to be started with a single command:
+```bash
+docker compose up
+```
+
+As part of the task, the following endpoints must be rewritten to work using SQL queries:
+
+- **Orders:**
+
+    - `POST /orders`: Create a new order.
+    - `GET /orders`: Retrieve all orders.
+    - `GET /orders/{id}`: Retrieve a specific order by ID.
+    - `PUT /orders/{id}`: Update an existing order.
+    - `DELETE /orders/{id}`: Delete an order.
+    - `POST /orders/{id}/close`: Close an order.
+- **Menu Items:**
+
+    - `POST /menu`: Add a new menu item.
+    - `GET /menu`: Retrieve all menu items.
+    - `GET /menu/{id}`: Retrieve a specific menu item.
+    - `PUT /menu/{id}`: Update a menu item.
+    - `DELETE /menu/{id}`: Delete a menu item.
+- **Inventory:**
+
+    - `POST /inventory`: Add a new inventory item.
+    - `GET /inventory`: Retrieve all inventory items.
+    - `GET /inventory/{id}`: Retrieve a specific inventory item.
+    - `PUT /inventory/{id}`: Update an inventory item.
+    - `DELETE /inventory/{id}`: Delete an inventory item.
+- **Aggregations:**
+
+    - `GET /reports/total-sales`: Get the total sales amount.
+    - `GET /reports/popular-items`: Get a list of popular menu items.
+
+In addition to rewriting the existing endpoints, you must also develop the following new ones:
+#### 1. Number of ordered items
+`GET /numberOfOrderedItems?startDate={startDate}&endDate={endDate}`: Returns a list of ordered items and their quantities for a specified time period. If the `startDate` and `endDate` parameters are not provided, the endpoint should return data for the entire time span.
```

Comment:
guess it should be part of /orders group -> GET /orders/numberOfOrderedItems?startDate={startDate}&endDate={endDate}

or a part of /reports group